experimental/dds/tree2/src/class-tree/schemaFactory.ts

```diff
@@ -530,10 +593,15 @@ export class SchemaFactory<TScope extends string, TName extends number | string
 	 * Some related information in https://github.com/microsoft/TypeScript/issues/55758.
 	 *
 	 * Also be aware that code which relies on this tends to break VSCode's IntelliSense every time anything related to that code (even comments) is edited.
-	 * The command `TypeScript: Restart TS Server` should fix it.
+	 * Running the command `TypeScript: Restart TS Server` with the schema file focused should fix it.
 	 * Sometimes this does not work: closing all open files except the schema before running the command can help.
 	 * Real compile errors (for example elsewhere in the file) can also cause the IntelliSense to not work correctly ever after `TypeScript: Restart TS Server`.
-	 * Intellisense has also shown problems when schema files with recursive types are part of a cyclic file dependency. Splitting the schema into its own file with minimal dependencies can help with this.
+	 *
+	 * Intellisense has also shown problems when schema files with recursive types are part of a cyclic file dependency.
+	 * Splitting the schema into its own file with minimal dependencies can help with this.
+	 *
+	 * Ensure `"noImplicitAny": true` is set in the `tsconfig.json`.
+	 * Without it recursive types that are not working properly can infer `any` and give very non-type-safe results instead of erroring.
```

Comment:
Nit
```suggestion
	 * Without it recursive types that are not working properly - they can infer `any` and give very non-type-safe results instead of erroring.
```

Reply:
I think you missed the "that" earlier in this sentence, as I think your change makes it malformed.

Reply:
Oh, yes you're right. So this would be more appropriate:
```suggestion
	 * Without it, recursive types that are not working properly can infer `any` and give very non-type-safe results instead of erroring.
```